### PR TITLE
Add separate buffer for container data types

### PR DESCRIPTION
**Pull request by samisms**
Separating the buffer for container data types. Container log volumes are quite high and they were filling up the common buffer leading to queues getting backed up. Separated thew buffers to alleviate the problem.
@Microsoft/omsagent-devs @keikhara

### CODE REVIEW

File: installer/conf/container.conf [hunk 1]
```diff
@@ -64,3 +64,17 @@
...
 	type filter_container
 </filter>
+
+#separate buffer for containers
+<match oms.container.**>
+  type out_oms
+  log_level info
+  buffer_chunk_limit 20m
+  buffer_type file
+  buffer_path /var/opt/microsoft/omsagent/state/state/out_oms_docker*.buffer
```
Comment (lagalbra) [inline installer/conf/container.conf]: You say "state" twice. Have you tested this?

Comment (samisms) [inline installer/conf/container.conf]: image
 Yes, based on whatever path the omsagent uses right now.

Comment (lagalbra) [inline installer/conf/container.conf]: Will this conf file ever get onto a VM without a Container? Perhaps that machine's setup is different, but with release 1.3.1-15, I have no ...state/state folder:

`ls: cannot access /var/opt/microsoft/omsagent/state/state/: No such file or directory`

Comment (samisms) [inline installer/conf/container.conf]: Removed the hardcoded path , Tested by modifying omsadmin.sh to update the buffer path during install. Image shows container.conf contents after onboarding to a workspace
image
Will send out a separate PR for the omsadmin.sh

File: installer/conf/container.conf [hunk 2]
```diff
@@ -64,3 +64,17 @@
...
 	type filter_container
 </filter>
+
+#separate buffer for containers
+<match oms.container.**>
+  type out_oms
+  log_level info
+  buffer_chunk_limit 20m
+  buffer_type file
+  buffer_path %STATE_DIR_WS%/out_oms_docker*.buffer
```
Comment (lagalbra) [inline installer/conf/container.conf]: Where are you replacing this string? [omsadmin.sh](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/scripts/omsadmin.sh#L820) currently only replaces this string in a hardcoded set of files (search page for `update_path`).

Comment (samisms) [inline installer/conf/container.conf]: if [ -s $OMSAGENTD_DIR/container.conf ] ; then
      update_path $OMSAGENTD_DIR/container.conf
fi
This is the change in copy_omsagent_d_conf() in omsadmin.sh I tested.

Comment (lagalbra) [inline installer/conf/container.conf]: For reference, that is this PR: https://github.com/Microsoft/OMS-Agent-for-Linux/pull/420
